libs/blocks/merch/checkout-context.js

```diff
@@ -0,0 +1,30 @@
+const DEFAULT_WORKFLOW_STEP = 'email';
```

Comment:
defaults values are already provided in tacocat. Anything we pass should be non default values.

Reply:
Workflow Step default is not provided in Tacocat. This is something I wanted to discuss, shall I add it to the taco and remove it from here?
For workflowStep it does make sense.

However there probably should not be default value for client id, so we still will need one in the milo, wdyt?

Reply:
it is though:
![image](https://user-images.githubusercontent.com/330057/230100036-f4c06609-1c67-4145-89a0-cf4e08895898.png)
yes, `client_id` is **adobe_com** by default but I agree that it must be overriden in the consumers.

Reply:
as discussed in call, might get an error because of an empty value being passed. i am going to check & fix it.